src/Domain.Schema/Attributes/Examples/GuidExamplesAttribute.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace OMP.Connector.Domain.Schema.Attributes.Examples
+{
+    public class GuidExamplesAttribute : ExampleAttribute
```

Comment:
What is this needed for?

Reply:
It seems these 'example' attributes are used in quite a number of places in the schema project. The Base Message, Model and Source uses these example attributes (but not only here).